# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eject/dont-use-kernel-headers.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./Makefile.am.orig	2004-02-10 22:23:13.052195592 +0000
+++ ./Makefile.am	2004-02-10 22:27:38.094902976 +0000
@@ -4,7 +4,7 @@
 
 SUBDIRS = po
 
-AM_CPPFLAGS = -I/usr/src/linux -I/usr/src/linux/include -DDEFAULTDEVICE=\"$(DEFAULTDEVICE)\" -DGETOPTLONG
+AM_CPPFLAGS = -DDEFAULTDEVICE=\"$(DEFAULTDEVICE)\" -DGETOPTLONG
 AM_CFLAGS = -Wall -O3 $(RPM_OPT_FLAGS)
 
 man_MANS = eject.1 volname.1
